terraform/modules/emr/templates/emr/get_scripts.sh

```diff
@@ -0,0 +1,30 @@
+SOURCE_LOCATION=$1
+DESTINATION_LOCATION=$2
+
+BUCKET="${config_bucket}"
+
+SOURCE_PATH=$BUCKET/$SOURCE_LOCATION
+
+(
+    # Import the logging functions
+    source /opt/emr/logging.sh
+
+    function log_wrapper_message() {
+        log_message "$${1}" "get_scripts.sh" "$${PID}" "$${@:2}" "Running as: ,$USER"
+    }
+    echo "Start Downloading files from $SOURCE_LOCATION to $DESTINATION_LOCATION"
+    log_wrapper_message "Start Downloading files from $SOURCE_LOCATION to $DESTINATION_LOCATION"
+
+    aws s3 cp $SOURCE_PATH $DESTINATION_LOCATION --recursive
```

Comment:
Did you look at s3-dist-cp? That would parallelise the copy? Also do we want to do this copying only on the master node?